eahub/profiles/models.py

```diff
@@ -168,12 +180,15 @@ class Profile(models.Model):
     cause_areas = postgres_fields.ArrayField(
         enum.EnumField(CauseArea), blank=True, default=list
     )
+    cause_areas_other = models.TextField(null=True, blank=True)
```

Comment:
`CharField`s and `TextField`s should not set `null=True` except in special circumstances, which this isn't. In Django, empty fields of those types are generally represented in SQL as the empty string, not as `NULL`.

This change will require deleting your migration file and rerunning `makemigrations`.